api-reference/10 UI Components/dxTabs/1 Configuration/orientation.md

```diff
@@ -5,7 +5,11 @@ default: 'horizontal'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies Tabs orientation.
```

Comment:
The same comment applies as in the other member (singular vs plural in noun phrases). 

I would also use consistent capitalization.

```suggestion
Specifies tab orientation.
```